From d96728f621889a4e6a7ca57b41af8eca7c94ec70 Mon Sep 17 00:00:00 2001
From: Jannick <thirdedition@gmx.net>
Date: Sun, 22 Dec 2019 12:30:00 +0100
Subject: [PATCH] configure.ac: set shared object extension to 'dll'

This patch hardcodes the shared object extension to dll on Windows
platforms, i.e.whenever EXEEXT is '.exe'.
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index f353cea3..af3432cb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -458,10 +458,12 @@ AH_BOTTOM([#include "custom.h"])
 dnl Crude but small hack to make plug-ins work on Mac OS X
 dnl We should really use the libtool value for shrext_cmds, but that
 dnl is not available here, since we do not use libtool at the top level.
+AS_VAR_IF([EXEEXT],[.exe],[GAWKLIBEXT=dll],[
 case $acl_shlibext in
 dylib)	GAWKLIBEXT=so ;;	# MacOS uses .dylib for shared libraries, but libtool uses .so for modules
 *) GAWKLIBEXT=$acl_shlibext ;;
 esac
+])
 AC_SUBST(GAWKLIBEXT)
 
 AC_CONFIG_FILES(Makefile
-- 
2.24.1.windows.2

